### ISSUE TITLE
codeblocks which end on the same line as the code are not stripped

### ISSUE THREAD

**Issue by bmintz** [Gorialis/jishaku#17]
## problem step
```
jsk py
`​`​`py
1+1`​`​`
```
or
```
jsk py `​`​`py
1+1`​`​`
```

## expected result

2

## actual result

```
Traceback (most recent call last):
  File "<string>", line 1
    1+1`​``
       ^
SyntaxError: invalid syntax
```
## remarks

i'd fix this myself but i don't understand your regex lol

**Comment by Gorialis** [comment]
I'm surprised this constitutes as a codeblock by Discord's standards. I don't think GitHub lets you do this. Will fix, in any case.

**Comment by bmintz** [comment]
Fixed in 6f5dcb198f3bb236bb133b07ef626df5e374ff51.
